src/io/flutter/sdk/FlutterSettingsConfigurable.form

```diff
@@ -14,9 +14,6 @@
           <grid row="0" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="3" anchor="0" fill="3" indent="0" use-parent-layout="false"/>
         </constraints>
         <properties/>
-        <clientProperties>
```

Comment:
Oh!  Sure.  Sorry about that.  It'd be great to prevent the UI editor from doing this.

@alexander-doroshko : thoughts?